### PR TITLE
Update KfW Geodata Model Guidelines

### PR DESCRIPTION
**Pull request by Maja4Dev**
I completed my update of the general guidelines (= index) - taking into account the read-me in Excel (Extra-Chapter), the notes in the columns of the excel template as well as the general recommendations (beyond protected areas) of Balthasar's tutorial.

I also added a new recommendation on filling the GADM GID, Level and version in the "Project location identifier" column, which needs to be discussed. I also already mentioned Kobo Toolbox, because this should be ready during the next days.

And I mentioned the geodata collection ToR everywhere where information by the KfW counterpart may be required or desired. We can then keep all info to be provided by KfW in these ToR. I will upload the updated ToR also in Github after review (see Kanban Board).

### CODE REVIEW

**Comment by fretchen** [comment]
@irie-pepe

**Review by Maja4Dev — COMMENTED** [review]
This is fine with me.

**Comment by fretchen** [comment]
I moved all the comments from the `index.md` file into the fitting issue. This allows us now to clean out the comments from the markdown file and then break it into parts in a second step in #41 . You will therefore find a number of suggested changes, which are basically removing comments and clean up for acceptance.

File: docs/index.md [hunk 1]
```diff
@@ -2,147 +2,112 @@
 comments: true
 ---
 
-# The KfW Geodata Model
+# Extra Page: Introduction
```
Comment (fretchen) [inline docs/index.md:5]:
```suggestion
# Introduction
```

File: docs/index.md [hunk 2]
```diff
@@ -2,147 +2,112 @@
...
+-   an exact project location is not to be collected or communicated due to security reasons -> then choose option: *approximate (security)*   
 
-Mixing the administrative unit levels is not allowed because this would render the geo-data Model inconsistent.
+-   the target location(s) is/are one or more administrative units, e.g. a district, a province or the entire country -> then choose option: *approximate (admin unit)*   
 
-The GIS database of GADM can be downloaded [here](https://gadm.org/data.html) and analyzed in a common GIS software such as QGIS, ArcGIS, or Google Earth Pro. Data collectors have to enter the GADM-defined polygon Geographic ID (GID), the GID level, and the utilized GADM database version. See Figure 2 below for more information:
+## Figures (still need to be decided)
```
Comment (fretchen) [inline docs/index.md:93]: Then please decide soon. I see two options:

1.) Open an issue where to fix this.
2.) Leave it in for the moment.

File: docs/index.md [hunk 3]
```diff
@@ -176,10 +141,53 @@ Below, the required project location data is defined for all FC projects. Projec
...
+### My KML file is too large to be sent via email. How can I send it to you? 
+In this case you should ask your KfW project counterpart. They might provide you with a fileshare (idgard) or you might be able to use your own system of choice. 
+
+... to be completed at the very end. 
+
+# Extra Page: Project Location Icons
+
+# Extra Page: Additional information material
+
+... Freds Tutorial
```
Comment (fretchen) [inline docs/index.md:193]:
```suggestion
```

File: docs/index.md [hunk 4]
```diff
@@ -176,10 +141,53 @@ Below, the required project location data is defined for all FC projects. Projec
...
+We use Google Earth Pro to verify project location data submitted to our teams which works best with KML. If you are not able to provide data in KML, you might also use GPKG or GEOJSON. We encourage you not to use the Shapefile format since the world has suffered enough from it ;) .
+
+### My KML file is too large to be sent via email. How can I send it to you? 
+In this case you should ask your KfW project counterpart. They might provide you with a fileshare (idgard) or you might be able to use your own system of choice. 
+
+... to be completed at the very end. 
+
+# Extra Page: Project Location Icons
+
+# Extra Page: Additional information material
```
Comment (fretchen) [inline docs/index.md:191]:
```suggestion
```

Removed as now noted in #41

File: docs/index.md [hunk 5]
```diff
@@ -176,10 +141,53 @@ Below, the required project location data is defined for all FC projects. Projec
...
+
+### Can I also use other geospatial formats rather than KML?
+We use Google Earth Pro to verify project location data submitted to our teams which works best with KML. If you are not able to provide data in KML, you might also use GPKG or GEOJSON. We encourage you not to use the Shapefile format since the world has suffered enough from it ;) .
+
+### My KML file is too large to be sent via email. How can I send it to you? 
+In this case you should ask your KfW project counterpart. They might provide you with a fileshare (idgard) or you might be able to use your own system of choice. 
+
+... to be completed at the very end. 
+
+# Extra Page: Project Location Icons
```
Comment (fretchen) [inline docs/index.md:189]:
```suggestion
```
Removed as now in #41

File: docs/index.md [hunk 6]
```diff
@@ -176,10 +141,53 @@ Below, the required project location data is defined for all FC projects. Projec
...
+| X. Primary Key (as provided in KML file) | Decimal Number  | Input  | Yes, if kmldata is provided | Per location | Required only in case you provide additonal geolocation information in another KML file. The geometry in the KML file should possess the same primary key as an attribute as it is listed here.|
+
+
+# Extra Page: FAQs
```
Comment (fretchen) [inline docs/index.md:177]:
```suggestion
# FAQs
```

File: docs/index.md (hunk 2 above)
Comment (Maja4Dev) [reply docs/index.md:93]: 2) Leave the figures in, but delete the Text in 104 in any case.

File: docs/index.md (hunk 4 above)
Comment (Maja4Dev) [reply docs/index.md:191]: All these Extra Page: ... should become new pages. We need this as a reminder.

File: docs/index.md (hunk 5 above)
Comment (Maja4Dev) [reply docs/index.md:189]: Same here - this is a reminder about changing the name of the icons category name.

File: docs/index.md (hunk 4 above)
Comment (fretchen) [reply docs/index.md:191]: The reminder is in #41

File: docs/index.md (hunk 2 above)
Comment (fretchen) [reply docs/index.md:93]:
```suggestion
## Figures
```

**Pull request merged**